client/src/modules/templates/bhDebtorGroupHistory.tmpl.html

```diff
@@ -0,0 +1,16 @@
+<div class="panel panel-default">
+    <p><u><span translate>DEBTOR_GROUP.CHANGE</span></u></p>
+
+    <ul class="list-unstyled">
+      <li ng-repeat="change in $ctrl.groupChanges | orderBy:'-created_at'">
+        <i class="fa fa-clock-o"></i>
+        <b><span translate>FORM.LABELS.FROM</span></b> {{change.group_prev}}
```

Comment:
Let's make this into an arrow to make it super clear:  Something like
```html
{{change.group_prev}} &#8594; {{change.group_next}}
```
which will render &#8594; as the arrow.